changelogs/unreleased/8132-mpryc

```diff
@@ -0,0 +1 @@
+Random race condition in the restore with managed fields
```

Comment:
@mpryc changelog needs to have PR number, not issue number -- s/8132/8133/ in the filename.

Also, I'd rephrase the changelog to describe the fix rather than the bug (since it will appear in release notes):
"Fixed race condition for conflicts on patching managed fields" or soething like that.

Reply:
Changed.